src/charm.py

```diff
@@ -499,7 +499,9 @@ def _on_config_changed(self, _event: ConfigChangedEvent) -> None:
         self._refresh_firewall()
         try:
             self._event_timer.ensure_event_timer(
-                "reconcile-runners", self.config["reconcile-interval"]
+                event_name="reconcile-runners",
+                interval=int(self.config["reconcile-interval"]),
```

Comment:
The conversion to int and validation of the `reconicle-interval` configuration, will be done in a future PR where all configuration in the `charm.py` is refactor to `charm_state.py`.